dev/AppNotifications/AppNotificationManager.h

```diff
@@ -1,27 +1,49 @@
+    };
+
+    struct AppNotificationManager : AppNotificationManagerT<AppNotificationManager, IAppNotificationManagerInternal>
     {
         AppNotificationManager() = default;
 
         static winrt::Microsoft::Windows::AppNotifications::AppNotificationManager Default();
+        void Register();
         void Register(winrt::Microsoft::Windows::AppNotifications::AppNotificationActivationInfo const& details);
         void Unregister();
-        winrt::event_token AppNotificationActivated(winrt::Windows::Foundation::TypedEventHandler<winrt::Microsoft::Windows::AppNotifications::AppNotificationManager, winrt::Microsoft::Windows::AppNotifications::AppNotificationActivatedEventArgs> const& handler);
-        void AppNotificationActivated(winrt::event_token const& token) noexcept;
+        void UnregisterAll();
+        winrt::event_token NotificationInvoked(winrt::Windows::Foundation::TypedEventHandler<winrt::Microsoft::Windows::AppNotifications::AppNotificationManager, winrt::Microsoft::Windows::AppNotifications::AppNotificationActivatedEventArgs> const& handler);
+        void NotificationInvoked(winrt::event_token const& token) noexcept;
         void Show(winrt::Microsoft::Windows::AppNotifications::AppNotification const& notification);
-        winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressResult> UpdateProgressDataAsync(winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData const data, hstring const tag, hstring const group);
-        winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressResult> UpdateProgressDataAsync(winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData const data, hstring const tag);
-        winrt::Microsoft::Windows::AppNotifications::AppNotificationSetting Enablement();
+        winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressResult> UpdateAsync(winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData const data, hstring const tag, hstring const group);
+        winrt::Windows::Foundation::IAsyncOperation<winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressResult> UpdateAsync(winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData const data, hstring const tag);
+        winrt::Microsoft::Windows::AppNotifications::AppNotificationSetting Setting();
         winrt::Windows::Foundation::IAsyncAction RemoveByIdAsync(uint32_t notificationId);
-        winrt::Windows::Foundation::IAsyncAction RemoveWithTagAsync(hstring const tag);
-        winrt::Windows::Foundation::IAsyncAction RemoveWithTagGroupAsync(hstring const tag, hstring const group);
-        winrt::Windows::Foundation::IAsyncAction RemoveGroupAsync(hstring const group);
+        winrt::Windows::Foundation::IAsyncAction RemoveByTagAsync(hstring tag);
```

Comment:
Why are we losing the constness

Reply:
Good catch. Will fix.

Reply:
Wasn't on purpose. I promise :)

Reply:
I'll send a new PR out addressing this in a bit.